fuse/dl/models/model_multihead.py

```diff
@@ -33,8 +33,8 @@ def __init__(
         self,
         backbone: torch.nn.Module,
         heads: Sequence[torch.nn.Module],
-        conv_inputs: Tuple[Tuple[str, int], ...] = None,
-        backbone_args: Tuple[Tuple[str, int], ...] = None,
+        conv_inputs: Tuple[Tuple[str, int], str] = None,
```

Comment:
It should be:
Union[Sequence[str], Sequence[Tuple[str, int]]

Reply:
or `Union[Tuple[str], Tuple[Tuple[str, int]]`, no? if we want it to precisely extend the current version

Reply:
oh actually I see why not, because we then want to reassign it and tuple is immutable